staging/src/k8s.io/client-go/rest/transport.go

```diff
@@ -118,3 +128,45 @@ func (c *Config) TransportConfig() (*transport.Config, error) {
 func (c *Config) Wrap(fn transport.WrapperFunc) {
 	c.WrapTransport = transport.Wrappers(c.WrapTransport, fn)
 }
+
+var CertificateCacheExpiryPeriod = time.Second
```

Comment:
reading top to bottom, godoc please

Reply:
> reading top to bottom, godoc please

not used anywhere else, don't make it a const like this, just bring it into the method.

Reply:
I was thinking maybe tests would want to edit, but yeah not particuarly useful